assets/js/theme/custom/order-product-data.js

```diff
@@ -0,0 +1,55 @@
+/* @TODO: Need to figure out a way to hide API TOKEN on the front. */
```

Comment:
@aleksandraworhacz The Storefront API token coming from BigCommerce is something that was designed to be available in the frontend. I don't think there is a way to hide it in the frontend.